bindings/go/README.md

```diff
@@ -62,5 +62,8 @@ cd motr/bindings/go/mcp && go build && go install
 The binary will be installed to your `GOBIN` directory
 (check with `go env GOBIN`).
```

Comment:
Just add here: "or `echo $(go env GOPATH)/bin`" and that will be enough.

Reply:
Thanks Andriy! I've updated the commit